staging/src/kubevirt.io/api/export/v1alpha1/types.go

```diff
@@ -127,6 +127,13 @@ type VirtualMachineExportLink struct {
 	// +listMapKey=name
 	// +optional
 	Volumes []VirtualMachineExportVolume `json:"volumes"`
+
+	// DefinitionUrl is the url that contains the VM definition in either JSON or YAML format
+	// +optional
+	DefinitionUrl string `json:"definitionUrl,omitempty"`
+
+	// CDIHeaderSecretUrl returns a Containerized Data Importer compatible secret
+	CDIHeaderSecretUrl string `json:"cdiHeaderSecretUrl,omitempty"`
```

Comment:
Wonder if there is a better/more extensible way to name/organize these params.  maybe have a `manifests` section like `volumes`?

```yaml
manifests:
- all
   https://virt-export-export-test.default.svc/internal/manifests/all
- auth-header-secret
   https://virt-export-export-test.default.svc/internal/manifests/secret
```

Reply:
My example above was bad.  This is what I meant:

```yaml
manifests:
- name: all
   url: https://virt-export-export-test.default.svc/internal/manifests/all
- name: auth-header-secret
   url:  https://virt-export-export-test.default.svc/internal/manifests/secret
```

Could maybe even add a `description` field in there as well.

I like this format because it is more extensible and in line with k8s api standards:

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#lists-of-named-subobjects-preferred-over-maps

Reply:
I went back and forth, on this, I went with the map since it matched what you asked for better, but easy enough to switch.